test-requirements.txt

```diff
@@ -5,7 +5,8 @@ flake8-bugbear; python_version >= '3.5'
 flake8-pyi>=20.5; python_version >= '3.6'
 lxml>=4.4.0
 psutil>=4.0
-pytest>=6.1.0,<7.0.0
+# pytest 6.2 does not support Python 3.5
+pytest>=6.1.0,<6.2.0
```

Comment:
Can’t you select pre-6.2 for Python 3.5 only?

Reply:
Oh, oops... My urge to get things green was all consuming and I merged right before you posted this, sorry :-(
We definitely could, although in the past we've had to make changes to mypy source code based on pytest type annotation changes, so things are a little simpler if we stick to a single pytest version for all of mypy. Hopefully we can drop Python 3.5 support soon anyway.

Reply:
I worry that this declaration will stay behind even after we drop 3.5, and once we try to catch up with pytest we’d have a bunch of issues to debug all at once. But it’s fine, we can cross that bridge when we get to it.